fdbclient/ServerKnobs.cpp

```diff
@@ -981,6 +981,9 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	// NOTE: 'token-name" can NOT contain '#' character
 	init( REST_KMS_CONNECTOR_VALIDATION_TOKEN_DETAILS,             "");
 
+	init( DYNAMIC_REPLICATION_ENABLED,                           false);
```

Comment:
Could you please group these new knobs with the ones related to DD, in the section between line 166 and 300.

Reply:
done